packages/core/src/conversation/InCallHandRaiseType.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Wire
+ * Copyright (C) 2024 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+/**
+ * true if the hand is raised, false if lowered
+ */
+export type InCallHandRaiseType = boolean;
```

Comment:
One file for one type 😟 
Do you think we should move all the types in the same file?

Reply:
I think we don't need it we can directly use boolean.

Reply:
done in https://github.com/wireapp/wire-web-packages/pull/6636/commits/59db7920db7ee8d294af0895a52eefc607f0fbd7